webpack/middleware/unauthenticated_response.js

```diff
@@ -0,0 +1,16 @@
+import {
+  LOG_OUT
+} from '../actions/types';
+
+const parentFromQuestions = store => next => action => {
+  let {payload} = action;
```

Comment:
Stylistic question: Any reason for doing this over `const payload = action.payload;'? If you are pulling multiple properties off of an object into variables in local scope, then I get it. But for just one, I find it harder to read.